__tests__/app.test.js

```diff
@@ -218,6 +218,25 @@ describe("GET", () => {
         });
     });
   });
+
+  //task 10
+  describe("/api/users", () => {
+    it("200: responds with an array of objects with 3 properties", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then((result) => {
+          expect(result.body.users.length).toBeGreaterThan(0);
+          result.body.users.forEach((user) => {
+            expect(user).toEqual({
```

Comment:
This might be a great use case for the matcher .toMatchObject! Have a look into it and see how you would implement it here :)